Formula/mame.rb

```diff
@@ -44,6 +44,13 @@ class Mame < Formula
   uses_from_macos "expat"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "gcc" # for C++17
```

Comment:
According [MAME's documents](https://docs.mamedev.org/initialsetup/compilingmame.html), it needs:
```suggestion
    depends_on "gcc" # for C++17
    depends_on "qt@5"
    depends_on "sdl2_ttf"
```

Reply:
Thanks for the information. The PR will be blocked by `qt@5` dependency as that formula needs to be fixed to build on Linux.